ozaria/site/styles/play/level/tome/spell-palette-view.sass

```diff
@@ -0,0 +1,249 @@
+      top: 80px
+      bottom: 0px
+      height: auto
+      width: $api-bar-width - 13
+      .section-header, .sub-section-header
+        display: block
+        color: white
+        margin-right: 10px
+
+      .spell-palette-thang-entry-view
+        display: inline-block
+        border: 5px solid black
+        border-image: url(/images/level/thang_avatar_frame.png) 20 20 20 20 fill round
+        margin: 1px
+
+        img
+          background-color: black
+          width: 72px
+          height: 72px
+          border: 1px solid transparent
+          cursor: pointer
+        &:hover img
+          border: 1px solid #f3cd90
+        &.selected img
+          border: 1px solid white
+          cursor: auto
+
+      .nano-slider
+        background-color: rgba(243, 169, 49, 0.5)
+    .right
+      position: absolute
+      right: 20px
+      min-width: 200px
+      left: $api-bar-width - 10px
+      top: 60px
+      bottom: 0px
+      //width: calc(100% - #{$api-bar-width + 50px})
+      @include box-shadow(0 0 0 #000)
+      .scrollArea
+        max-height: calc(100% - 20px)
+        width: 100%
+        border-style: solid
+        border-image: url(/images/level/popover_border_background.png) 16 12 fill round
+        border-width: 16px 12px
+        margin-top: 20px
+        margin-bottom: 20px
+        position: relative
+        overflow-y: auto
+
+      .always-scroll-y
+        overflow-y: scroll
+        .description
+          img
+            float: left
+
+      .closeBtn
+        position: absolute;
+        right: -10px;
+        top: 15px;
+        width: 40px;
+        height: 40px;
+        cursor: pointer;
+        font-size: 20px
+        z-index: 2;
+
+      h1:not(.not-code), h2:not(.not-code), h3:not(.not-code), h4:not(.not-code), h5:not(.not-code), h6:not(.not-code)
+        font-family: Monaco, Menlo, Ubuntu Mono, Consolas, "source-code-pro", monospace !important
+        font-variant: normal
+        color: purple
+        text-transform: auto
+
+        body[lang="he"] &, body[lang="ar"] &, body[lang="fa"] &, body[lang="ur"] &
+          font-family: "Cousine", "Courier", "Courier New", monospace !important
+
+      .popover-title
+        background-color: transparent
+        margin: 0 14px
+        padding: 8px 0
+        border-bottom-color: #ccc
+
+      .popover-content
+        max-height: 100%
+        overflow-y: auto
+        margin-right: 10px
+        img
+          float: right
+
+      &.top .arrow
+        bottom: -2%
+      &.bottom .arrow
+        top: -2%
+      &.left .arrow
+        right: 0%
+      &.right .arrow
+        left: -3%
+
+      .docs-ace-container
+        padding: 2px 4px
+        border-radius: 4px
+
+        &, .docs-ace
+          background-color: #f9f2f4
+          font-size: 12px
+          font-family: Monaco, Menlo, Ubuntu Mono, Consolas, "source-code-pro", monospace !important
+
+          body[lang="he"] &, body[lang="ar"] &, body[lang="fa"] &, body[lang="ur"] &
```

Comment:
Do we have standards or documentation anywhere regarding the Typography rules we follow between the various languages?
I.e.
 - Fonts for different sets of languages
 - Direction of text based on language

Non blocking and just thought of this as I read this.

Reply:
I am not sure if we have any such documentation, but will definitely be useful and something to think about for the future. 👍